modules/ROOT/pages/use-anypoint-monitoring.adoc

```diff
@@ -0,0 +1,115 @@
+
+* US control plane: `dias-ingestor-router.us-east-1.prod.cloudhub.io` 
+* EU control plane: `dias-ingestor-router.eu-central-1.prod-eu.msap.io`
+
+[NOTE]
+====
+If you are using a version of Anypoint Runtime Fabric older than 1.8.50, use port 5044 instead of port 443. Use the following domains:
+
+* US control plane: `dias-ingestor-nginx.prod.cloudhub.io` 
+* EU control plane: `dias-ingestor-nginx.prod-eu.msap.io`
+====
+
+== Metrics
+
+Each Mule app and API gateway replica runs a metrics agent that collects and transmits metrics data to 
+Anypoint Monitoring via secure mutual TLS encryption. Each replica can cache up to 100 megabytes of metrics, after which the metrics data is rotated to provide storage for more recent data. In the event that metrics cannot be sent to Anypoint Monitoring, this typically provides at least 4 days worth of storage per replica. Note that custom metrics might consume more storage.
+
+[NOTE]
+--
+The following Anypoint Runtime Fabric updates do not impact or restore monitoring metrics of applications deployed on Runtime Fabric:
+
+* A backup and restore of Runtime Fabric
+* An upgrade of Runtime Fabric
+* OS patching of Runtime Fabric nodes
+--
+
+== Logs 
+
+Anypoint Monitoring provides access to xref:monitoring::logs.adoc[log data] for applications deployed to Rutime Fabric. To access logs with Anypoint Monitoring, you need a https://www.mulesoft.com/anypoint-pricing[Titanium subscription]. 
+
+Each Mule app and API gateway replica can store up to 450 megabytes of log data on disk, after which log data 
+is rotated to provide storage for more recent log data.
+
+Logs generated by each replica of a Mule app and API gateway are collected and written to disk on the worker 
+node on which they are running. Each worker node runs a single log forwarding agent, which monitors this location 
+and transmits the log data to Anypoint Monitoring.
+
+[NOTE]
+====
+The log forwarding agent can transmit up to 300 kilobytes of log data per second. You should not deploy more than
+50 replicas per worker node.
+
+In some cases, logs with multiple lines might not be fully transmitted to Anypoint Monitoring. This can occur if 
+multiple lines of log output are split across two files due to log rotation inside Runtime Fabric.
+====
+
+You can enable log forwarding and set log levels when you xref:deploy-to-runtime-fabric.adoc#configure-logging-and-log-levels[deploy applications in Runtime Manager]. 
+
+=== Resource and Permission Requirements for Anypoint Monitoring Agent
+
+MuleSoft runs Anypoint Monitoring agents alongside Mule apps to send telemetry to Anypoint control plane.
+
+For Mule apps deployed or managed using Runtime Fabric agent version 2.5.x or lower, two different agents are used as follows:
+
+* One daemonset per underlying node to transporting logs to Anypoint monitoring control plane.
```

Comment:
```suggestion
* One daemonset per underlying node to transport logs to Anypoint monitoring control plane.
```

Reply:
I looked up daemonset because I hadn't heard of it and I saw that it is usually written like this: DaemonSet.